tests/contract_testcases/solana/error2.dot

```diff
@@ -0,0 +1,50 @@
+strict digraph "tests/contract_testcases/solana/error2.sol" {
+	X1 [label="name: X1\ntests/contract_testcases/solana/error2.sol:2:1-14\nvalue: a"]
+	X2 [label="name: X2\ntests/contract_testcases/solana/error2.sol:7:1-14\nvalue: b"]
+	X2_4 [label="name:X2\ntests/contract_testcases/solana/error2.sol:6:7-9\nfield name:bar ty:int256"]
+	X2_5 [label="name:X2\ncontract: 0\ntests/contract_testcases/solana/error2.sol:11:8-10\nfield name:bar ty:int256"]
+	X3 [label="name:X3\ncontract: 0\ntests/contract_testcases/solana/error2.sol:12:8-10\nfield name:X ty:int256"]
+	X4 [label="name:X4\ncontract: 0\ntests/contract_testcases/solana/error2.sol:13:8-10\nfield name:X ty:int256\nfield name: ty:bool"]
+	X5 [label="name:X5\ncontract: 0\ntests/contract_testcases/solana/error2.sol:14:8-10\nfield name: ty:unresolved"]
+	X6 [label="name:X6\ncontract: 0\ntests/contract_testcases/solana/error2.sol:15:8-10\nfield name: ty:unresolved"]
+	contract [label="contract C\ntests/contract_testcases/solana/error2.sol:10:1-18:2"]
+	diagnostic [label="X1 is already defined as an enum\nlevel Error\ntests/contract_testcases/solana/error2.sol:3:7-9"]
+	note [label="location of previous definition\ntests/contract_testcases/solana/error2.sol:2:6-8"]
+	diagnostic_15 [label="expecting 'error', got 'foo'\nlevel Error\ntests/contract_testcases/solana/error2.sol:4:1-4"]
+	diagnostic_16 [label="expression found where 'error' was expected\nlevel Error\ntests/contract_testcases/solana/error2.sol:5:1-4"]
+	diagnostic_17 [label="X2 is already defined as an error\nlevel Error\ntests/contract_testcases/solana/error2.sol:7:6-8"]
+	note_18 [label="location of previous definition\ntests/contract_testcases/solana/error2.sol:6:7-9"]
+	diagnostic_19 [label="found contract 'C'\nlevel Debug\ntests/contract_testcases/solana/error2.sol:10:1-18:2"]
+	diagnostic_20 [label="X2 is already defined as an error\nlevel Warning\ntests/contract_testcases/solana/error2.sol:11:8-10"]
+	note_21 [label="location of previous definition\ntests/contract_testcases/solana/error2.sol:6:7-9"]
+	diagnostic_22 [label="error 'X3' has duplicate field name 'X'\nlevel Error\ntests/contract_testcases/solana/error2.sol:12:23-24"]
+	note_23 [label="location of previous declaration of 'X'\ntests/contract_testcases/solana/error2.sol:12:11-16"]
+	diagnostic_24 [label="mapping type is not permitted as error field\nlevel Error\ntests/contract_testcases/solana/error2.sol:14:11-32"]
+	diagnostic_25 [label="type 'X' not found\nlevel Error\ntests/contract_testcases/solana/error2.sol:15:11-12"]
+	diagnostic_26 [label="expecting 'error', got 'meh'\nlevel Error\ntests/contract_testcases/solana/error2.sol:16:2-5"]
+	diagnostic_27 [label="expression found where 'error' was expected\nlevel Error\ntests/contract_testcases/solana/error2.sol:17:2-6"]
```

Comment:
I think this message is misleading. Can't we have anything other than an error after an error definition?